osu.Game.Tournament/Screens/Board/BoardScreen.cs

```diff
@@ -741,6 +723,7 @@ public bool DetectWin()
                     CurrentMatch.Value.Team2Score.Value = 6;
                     CurrentMatch.Value.Team1Score.Value = 0;
                 }
+
                 return true;
             }
```

Comment:
这里的这段代码理论上永远不会运行

晚些时候我会过一遍整个逻辑

Reply:
But this did worked...
This will give the detected winner a maximum score, making it the *actual* winner in terms of score, and let it be shown on the Winner view.

但是运行时确实是有用的
这段代码主要是根据获胜方，给它赋最高分，从而在胜者视图显示